src/data/pokemon/base_stats.h

```diff
@@ -308,7 +308,9 @@ const struct BaseStats gBaseStats[] =
         .expYield = 178,
         .evYield_SpAttack  = 2,
         .evYield_SpDefense = 1,
-        .item2 = ITEM_NONE, // ITEM_SILVER_POWDER,
+        #ifdef ITEM_EXPANSION
+            .item2 = ITEM_SILVER_POWDER,
+        #endif
```

Comment:
I feel that that `ITEM_EXPANSION` should be replaced by the corresponding item, like this:
```c
        #ifdef ITEM_SILVER_POWDER
            .item2 = ITEM_SILVER_POWDER,
        #endif
```